Sources/RoktUXHelper/Data/Model/ExperienceResponse/CatalogItem.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CatalogItem.swift
+//  RoktUXHelper
+//
+//  Licensed under the Rokt Software Development Kit (SDK) Terms of Use
+//  Version 2.0 (the "License");
+//
+//  You may not use this file except in compliance with the License.
+//
+//  You may obtain a copy of the License at https://rokt.com/sdk-license-2-0/
+
+import Foundation
+
+struct CatalogItem: Codable {
```

Comment:
We probably need `ProviderData` serialized too
https://github.com/ROKT/cart-api/blob/6f4241c2dc1320b3ee3240d79a01bb882f1c8a7a/cart-api/src/Rokt.Cart.App.Interfaces/Pocos/CatalogItem.cs#L33C9-L33C104

Reply:
Yes I will add it in my PR when we support the partner event.

Reply:
Danial will be adding on to catalog items in his PR.